apps/web/src/pages/templates/components/in-app-editor/InAppContentCard.tsx

```diff
@@ -70,49 +58,14 @@ export function InAppContentCard({ openVariablesModal }: { openVariablesModal: (
         radius={'xl'}
       />
       <When truthy={activeTab === PREVIEW}>
-        <Grid mt={24} mb={0}>
-          <Grid.Col span={9} p={0}>
-            <div style={{ margin: '0 10px' }}>
-              <InAppEditorBlock payload={payloadValue} readonly={true} preview={true} />
```

Comment:
I initially thought to use that component but saw it is being used for editing and previewing, which was ok previously because the editor and preview screen were the same in designs.
Now that the design varies a lot for editor and preview. Handling cases like overlay blur will be a little complex; let's separate those into two different components instead of a single component handling editing and previewing. This will also help us make changes to any designs for the editor or preview easier in the future.
I plan to re-use the avatar and action buttons components in the preview instead of new components for those.